### PR TITLE
Get types function

### PR DESCRIPTION
**Pull request by ritisha2000**

### CODE REVIEW

File: R/pokehelperr.R [hunk 1]
```diff
@@ -1,3 +1,6 @@
+library(glue)
+library(dplyr)
+library(stringr)
```
Comment (J99thoms) [inline R/pokehelperr.R:3]: When making a package, we shouldn't import packages in this manner. We need to add them to the package's dependencies via use_package(<package_name>).

File: R/pokehelperr.R [hunk 2]
```diff
@@ -20,7 +23,33 @@
 #' get_types(list('Pikachu', 'Eevee', 'Charizard', ...))
 #'
 get_types <- function(pokemon_names) {
-  # Function code (TBD in Milestone 3)
+   url <- "https://raw.githubusercontent.com/UBC-MDS/pokehelpyer/main/data/pokemon.csv"
```
Comment (J99thoms) [inline R/pokehelperr.R:26]: It is probably best to use a local copy of the data frame to improve efficiency. For example, see https://github.com/UBC-MDS/pokehelperr/pull/21/files#r1089860169.

File: R/pokehelperr.R [hunk 3]
```diff
@@ -20,7 +23,33 @@
...
+        name <- trim(name)
+        name <- tolower(name)
+
+        if (!(name %in% name_with_symbol)){
+            name <- str_remove(name, "[^a-zA-Z0-9 ]")
+        }
+
+        type <- names_types_df |> 
+            filter(Name == name) |>
+            select(`Type.1`, `Type.2`)
```
Comment (J99thoms) [inline R/pokehelperr.R:46]: Whenever you use functions from another package, you need to namespace them, e.g. `dplyr::filter` and `dplyr::select`

**Review by J99thoms — CHANGES_REQUESTED** [review]

File: R/pokehelperr.R (hunk 1 above)
Comment (J99thoms) [reply R/pokehelperr.R:3]: See https://r-pkgs.org/whole-game.html#use_package.

File: R/pokehelperr.R (hunk 3 above)
Comment (J99thoms) [reply R/pokehelperr.R:46]: See https://r-pkgs.org/whole-game.html#use_package.

File: R/pokehelperr.R (hunk 1 above)
Comment (ritisha2000) [reply R/pokehelperr.R:3]: Oh sorry, forgot to remove them. Thanks

**Review by AguilarRaul — APPROVED** [review]
Looks great

**Pull request closed without merging**